docs/contributor-docs/style-guide.md

```diff
@@ -35,7 +38,18 @@ Sections within a section that is specific to a certain type of circumstance sho
 
 ---
 
+## Bullets and Indentations
+### For example:
+Mkdocs is finicky and requires double indentations to create properly indented sub-bullets.
+
+* Item A
+    * Item B
+    	* Item C
+
+---
+
 ## Terms We Use
+- personal computer (**not** personal computer, personal computer, or tablet, etc.)
```

Comment:
I'm not sure I understand this line -- can you clarify this?

Reply:
Thanks for finding this. It's now fixed.